doc/stolonctl.md

```diff
@@ -1,18 +1,39 @@
 ## Stolon Client (stolonctl)
 
-`stolonctl` is the stolon client which controls the stolon cluster(s)
+`stolonctl` is the stolon client which controls the stolon cluster(s) and may be executed inside a pod running a stolon component or externally. 
 
-It needs to communicate with the cluster store (providing `--store-backend` and related option) on which the requested cluster name (`--cluster-name`) is running.
+Since `stolonctl`needs to communicate with the cluster store, it requires providing the requested cluster name (`--cluster-name`), its store backend (`--store-backend`), and when using "kubernetes" as store the kind of kubernetes resources (`--kube-resource-kind`).
 
-To avoid repeating for every command (or inside scripts) all the options you can export them as environment variables. Their name will be the same as the option name converted in uppercase, with `_` replacing `-` and prefixed with `STOLONCTL_`.
+Example, check stolon cluster status for a cluster named "blue-stolon" using "etcdv3" as a store:
+```
+$ stolonctl status --cluster-name=blue-stolon --store-backend=etcdv3 --store-endpoints=http://etcd-0.etcd:2379,http://etcd-1.etcd:2379,http://etcd-2.etcd:2379
```

Comment:
to be consistent with other examples in the doc and as generic as possible I'll change it to:

```
stolonctl status --cluster-name=stolon-cluster --store-backend=etcdv3 --store-endpoints=http://etcd-0:2379,http://etcd-1:2379,http://etcd-2:2379
```